packages/react-cosmos-webpack/src/export.js

```diff
@@ -77,8 +77,16 @@ module.exports = function startExport() {
   // Copy static files first, so that the built index.html overrides the its
   // template file (in case the static assets are served from the root path)
   if (publicPath) {
-    const exportPublicPath = path.join(outputPath, publicUrl);
-    fs.copySync(publicPath, exportPublicPath);
+    if (outputPath.indexOf(publicPath) === -1) {
```

Comment:
I'm not sure if this will solve the case where `publicPath` is `./`. Might need to use `path.resolve` first, or `path.join`.

Anyway, check my [comment](https://github.com/react-cosmos/react-cosmos/issues/415#issuecomment-325118975) surrounding `publicPath`. Maybe clearer docs would be better than this guard.

Reply:
The paths are resolved and absolute at this point, so it works. This is the output you've gotten in react-test-buffet with this patch:

![image](https://user-images.githubusercontent.com/250750/29741170-563d1658-8a67-11e7-8b11-edf5adc43c35.png)

Reply:
The docs already describe `publicPath` as

> Where to serve static files from. Like --content-base in webpack-dev-server.

But any suggestion is welcome!

Reply:
Ah, sweet.